crates/uv/src/commands/python/uninstall.rs

```diff
@@ -278,28 +279,27 @@ async fn do_uninstall(
     }
 
     // Report on any uninstalled installations.
-    if !uninstalled.is_empty() {
-        if let [uninstalled] = uninstalled.as_slice() {
+    if let Some(first_uninstalled) = uninstalled.first() {
```

Comment:
The destructuring approach no longer works now that this is an `IndexSet` instead of a `Vec`.